### PR TITLE
Standardize UG

### PR DESCRIPTION
**Pull request by alvintfl**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -353,21 +353,29 @@ Examples:
...
 
-* INDEX must be a positive integer more than 0.
+<div markdown="block" class="alert alert-warning">
 
-* At least one of the optional fields must be provided.
+:warning:There will be an error if you:<br>
 
-* Input values will replace existing values.
+* Do not adhere to [field constraints](#section-2-tasks).<br> 
+
```
Comment (feliciagan) [inline docs/UserGuide.md:369]: Add "Edit the task such that it becomes a duplicate task to one already in the task list. Two tasks are the same if they have the same description, deadline and labels."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -353,21 +353,33 @@ Examples:
...
-* Index of a task is its index number on the task list.
+* Index of a task is its index number on the currently shown task list.
+ 
+* Input values will replace existing values.
 
-* INDEX must be a positive integer more than 0.
+<div markdown="block" class="alert alert-warning">
 
-* At least one of the optional fields must be provided.
+:warning:There will be an error if you:<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: space btwn icon and words
ie
:warning: There will be

File: docs/UserGuide.md [hunk 3]
```diff
@@ -620,16 +632,21 @@ Example:
...
-* The label is also deleted.
+Format: `deleteA LABEL_NAME [MORE_LABEL_NAMES]...`
 
 * If a contact/task has multiple labels, it will not be deleted as long as it has at least one label.
   Instead, the labels will be removed from the contact/task.
 
-<div markdown="span" class="alert alert-warning">:warning: Label does not exist.
+<div markdown="block" class="alert alert-warning">
+
+:warning:There will be an error if you:<br>
```
Comment (feliciagan) [inline docs/UserGuide.md]: same thing

**Comment by junwei-tan** [comment]
LGTM

**Pull request merged**